tools/data-handler/src/remove.ts

```diff
@@ -45,22 +42,26 @@ export class Remove extends EventEmitter {
 
     // Removes card from project or template
     private async removeCard(cardKey: string) {
-        const cardFolder = await Remove.project.cardFolder(cardKey);
-        if (!cardFolder) {
-            throw new Error(`Card '${cardKey}' not found`);
-        }
-
-        // Imported templates cannot be modified.
-        if (cardFolder.includes(`${sep}modules${sep}`)) {
-            throw new Error(`Cannot modify imported module`);
-        }
-
-        // Calculations need to be updated before card is removed.
-        const card = await Remove.project.findSpecificCard(cardKey);
-        if (card) {
-            this.emit('removed', card);
-        }
-        await deleteDir(cardFolder);
+      const cardFolder = await Remove.project.cardFolder(cardKey);
+      if (!cardFolder) {
+        throw new Error(`Card '${cardKey}' not found`);
+      }
+
+      // Imported templates cannot be modified.
+      if (cardFolder.includes(`${sep}modules${sep}`)) {
+        throw new Error(`Cannot modify imported module`);
+      }
+
+      // Calculations need to be updated before card is removed.
+      const card = await Remove.project.findSpecificCard(cardKey);
```

Comment:
This is fine for now; I think I will at some stage make the `handleDeleteCard` to call 
```
        await this.generateCardTreeContent(undefined);
        await this.genereteCardTree();
```
 (like in adding new cards case)
 It is almost as fast to generate the whole calculation files from scratch than to edit the existing ones (tested with ~22000 cards). Then we don't need to call `getCards(...)` in `handleDeletedCard` which is the cause of the issue.